geoutils/georaster/raster.py

```diff
@@ -895,7 +901,7 @@ def info(self, stats: bool = False) -> str:
             f"Size:                 {self.width}, {self.height}\n",
             f"Number of bands:      {self.count:d}\n",
             f"Data types:           {self.dtypes}\n",
-            f"Coordinate System:    EPSG:{self.crs.to_epsg()}\n",
+            f"Coordinate System:    {self.crs}\n",
```

Comment:
But `self.CRS` is a rasterio object, so it does not really print nicely... If epsg does not exist, we should use `self.crs.to_string`.
If `self.crs` is not always a rasterio CRS (not so great...) then add a condition.

Reply:
Done with a condition